Tasks/Maven/pmdForMaven.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference path='../../definitions/vsts-task-lib.d.ts' />
+
+import tl = require('vsts-task-lib/task');
+import trm = require('vsts-task-lib/toolrunner');
+
+// Adds PMD goals, if selected by the user
+export function applyPmdArgs(mvnRun: trm.ToolRunner):void {
```

Comment:
It doesn't seem worth the effort of having a separate file for 1-line method, unless you think this file will grow with lots of pmd related functionality.